resolvers/deno/Cargo.toml

```diff
@@ -25,6 +25,7 @@ deno_package_json.features = ["sync"]
 deno_path_util.workspace = true
 deno_semver.workspace = true
 node_resolver.workspace = true
+node_resolver.features = ["sync"]
```

Comment:
Ideally we'd have this crate work with sync and not sync, but I haven't had time to make it do that yet.